### PR TITLE
Show status message in evaluation job viewer when error occurred

### PR DESCRIPTION
**Pull request by gentlecat**
*Related to [AB-61](http://tickets.musicbrainz.org/browse/AB-61).* :boom:

### CODE REVIEW

File: webserver/static/js/eval-jobs-viewer.js [hunk 1]
```diff
@@ -222,8 +224,16 @@ var JobDetails = React.createClass({
...
+                var errorMsg = "";
+                if (this.props.statusMsg) {
+                    errorMsg = <p>
+                        Error details:<br />
+                        {this.props.statusMsg}
+                    </p>
+                }
                 status = <div className="alert alert-danger">
-                    This evaluation job has failed!
+                    <strong>This evaluation job has failed!</strong>
```
Comment (alastair) [inline webserver/static/js/eval-jobs-viewer.js]: We don't need a ! here, but otherwise fine.

**Comment by alastair** [comment]
cool :car:

File: webserver/static/js/eval-jobs-viewer.js (hunk 1 above)
Comment (gentlecat) [inline webserver/static/js/eval-jobs-viewer.js]: `.`?

**Comment by gentlecat** [comment]
:blue_car:

**Pull request merged**